flamingpy/codes/surface_code.py

```diff
@@ -398,9 +378,65 @@ def identify_stabilizers(self):
                 [self.graph.to_indices[point] for point in syndrome_coords],
             )
             setattr(self, ec + "_stabilizers", all_cubes)
-        for att in ["_syndrome_inds", "_syndrome_coords"]:
-            new_attr = sum((getattr(self, ec + att) for ec in self.ec), start=[])
-            setattr(self, "all" + att, new_attr)
+
+    def periodic_boundary_stabilizers(self, ec, stabe, actual_stabe):
+        """Dealing with stabilizers at periodic boundaries.
+
+        Note this method is directly modifying the reference of
+        `actual_stabe`.
```

Comment:
```suggestion
        ``actual_stabe``.
```

Reply:
@thisac, thanks for fixing many typos and formatting errors in this PR.